charts/harvester-csi-driver-lvm/templates/csi.yaml

```diff
@@ -0,0 +1,164 @@
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: harvester-csi-driver-lvm-plugin
+  labels:
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+spec:
+  revisionHistoryLimit: 10
+  selector:
+    matchLabels:
+      app: harvester-csi-driver-lvm-plugin
+  template:
+    metadata:
+      labels:
+        app: harvester-csi-driver-lvm-plugin
+    spec:
+      serviceAccountName: harvester-csi-driver-lvm
+{{- if .Values.tolerations.plugin }}
+      tolerations:
+{{ toYaml .Values.tolerations.plugin | indent 8 }}
+{{- end }}
+{{- if .Values.nodeSelector.plugin }}
+      nodeSelector:
+{{ toYaml .Values.nodeSelector.plugin | indent 8 }}
+{{- end }}
+      containers:
+      - name: node-driver-registrar
+        args:
+        - --v=5
+        - --csi-address=/csi/csi.sock
+        - --kubelet-registration-path={{ .Values.kubernetes.kubeletPath }}/plugins/{{ .Values.lvm.driverName }}/csi.sock
+        env:
+        - name: KUBE_NODE_NAME
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: spec.nodeName
+        image: {{ template "externalImages.csiNodeDriverRegistrar" . }}
+        imagePullPolicy: IfNotPresent
+        resources: {}
+        securityContext:
+          privileged: true
+        terminationMessagePath: /dev/termination-log
+        terminationMessagePolicy: File
+        volumeMounts:
+        - mountPath: /csi
+          name: socket-dir
+        - mountPath: /registration
+          name: registration-dir
+      - name: harvester-csi-driver-lvm-plugin
+        args:
+        - --drivername={{ .Values.lvm.driverName }}
+        - --endpoint=unix:///csi/csi.sock
+        - --hostwritepath={{ .Values.lvm.hostWritePath }}
+        - --nodeid=$(KUBE_NODE_NAME)
+        - --namespace={{ .Release.Namespace }}
+        - --provisionerimage={{ .Values.provisionerImage.repository }}:{{ .Values.provisionerImage.tag }}
+        - --pullpolicy={{ .Values.provisionerImage.pullPolicy }}
+        env:
+        - name: KUBE_NODE_NAME
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: spec.nodeName
+        image: "{{ .Values.pluginImage.repository }}:{{ .Values.pluginImage.tag }}"
+        imagePullPolicy: {{ .Values.pluginImage.pullPolicy }}
+        livenessProbe:
+          failureThreshold: 5
+          httpGet:
+            path: /healthz
+            port: healthz
+            scheme: HTTP
+          initialDelaySeconds: 10
+          periodSeconds: 2
+          successThreshold: 1
+          timeoutSeconds: 3
+        ports:
+        - containerPort: 9898
+          name: healthz
+          protocol: TCP
+        resources: {}
+        securityContext:
+          privileged: true
+        terminationMessagePath: /termination.log
```

Comment:
`terminationMessagePath` for node-driver-registrar and liveness-probe is `/dev/termination-log`, but this one for harvester-csi-driver-lvm-plugin is `/termination.log`. Should they all be the same?

Reply:
Ditto. I thought that was the customized path, so we could keep it or make some changes in the next update.